usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -225,3 +229,29 @@ def concat_if_array(data):
             # This should never happen if TinyShield is functioning properly
             logger.error("Keyword submitted was not a string or array")
             return ""
+
+
+def get_number_of_unique_terms(filter_query: ES_Q, field: str) -> int:
+    search = TransactionSearch().filter(filter_query)
+    cardinality_aggregation = A("cardinality", field=field)
+    search.aggs.metric("field_count", cardinality_aggregation)
+    response = search.handle_execute()
+    response_dict = response.aggs.to_dict()
+    return response_dict.get("field_count", {"value": 0})["value"]
+
+
+def get_sum_aggregations(field_to_sum: str, pagination: Pagination) -> Dict[str, A]:
```

Comment:
Could this title be more specific to its reusable use case? Not a rhetorical question, I honestly don't know.

Reply:
Also couldn't think of a better name. Ideally this could be used to generate the sum aggregations for any Elasticsearch query that is using the DSL.